tests/with_synth/test_aggregates.py

```diff
@@ -0,0 +1,33 @@
+from openfisca_uk_data import SynthFRS
+from openfisca_uk import Microsimulation
+import pytest
+
+sim = Microsimulation(dataset=SynthFRS)
+
+average_tests = (("in_poverty_bhc", 0.15, 0.1),)
```

Comment:
could you add a comment indicating that the 0.1 is the absolute window rather than relative? (also this seems high)

Reply:
Yeah it is rather high unfortunately since the synthetic dataset isn't so close here. Shuffling fields doesn't affect individual variable aggregates like tax so much but does change household and family aggregates a fair amount.